Makefile

```diff
@@ -229,6 +229,9 @@ endif
 update:
 	@$(MAKE) shell CMD="-c 'hack/update-all.sh'"
 
+update-crd:
```

Comment:
CRDs are also updated in the `update` target. Not sure why adding a new target to just update CRD.

Reply:
This is for developing usage only, sometimes when we change some of the CRDs, we need to generate the new yaml and CRD.go to call `make-update`, but the entire `make-update` is very slow, so this `make update-crd `may be a shortcut in this scenario.

Reply:
How about adding a comment to explain this is for development only?

Reply:
Done, added the comment.

Reply:
Make update is now much faster :)

Reply:
@kaovilai 
Yes, I noticed this recently, thanks for making this improvement.
For `make update-crd`, let's keep it since it is not harmful and it does things less and help developers to save time.